Examples/MAX32655/Coremark/README.md

```diff
@@ -14,17 +14,19 @@ Universal instructions on building, flashing, and debugging this project can be
 
 ### Project-Specific Build Notes
 
-To comply with the CoreMark rules, the only source files which are included in this example directory are the core_portme.c/.h files, the rest (including main) are located in the [Coremark](../../../Libraries/Coremark/) library.
+* To comply with the CoreMark rules, the only source files which are included in this example directory are the core_portme.c/.h files, the rest (including main) are located in the [Coremark](../../../Libraries/Coremark/) library.
```

Comment:
Very minor, but I think it should be `CoreMark` (it's a registered trademark). We goofed when we added the library and named it `Coremark`. We can't change the directory name, but we can start naming it properly going forward.

Reply:
I made a note to make that change. Since it affects multiple parts, I'll do that in a separate PR.